tests/test_bulk_tx_basic/src/main.xc

```diff
@@ -13,59 +12,28 @@
 #define EP_COUNT_OUT   		(6)
 #define EP_COUNT_IN    		(6)
 
-/* Endpoint type tables */
+#include "shared.h"
+
 XUD_EpType epTypeTableOut[EP_COUNT_OUT] = {XUD_EPTYPE_CTL,
                                                 XUD_EPTYPE_BUL,
-                                                 XUD_EPTYPE_BUL,
-                                                 XUD_EPTYPE_BUL,
-                                                 XUD_EPTYPE_BUL,
-                                                 XUD_EPTYPE_BUL};
+                                                XUD_EPTYPE_BUL,
+                                                XUD_EPTYPE_BUL,
+                                                XUD_EPTYPE_BUL,
+                                                XUD_EPTYPE_BUL};
 XUD_EpType epTypeTableIn[EP_COUNT_IN] =   {XUD_EPTYPE_CTL, 
                                                 XUD_EPTYPE_BUL,
                                                 XUD_EPTYPE_BUL,
                                                 XUD_EPTYPE_BUL,
                                                 XUD_EPTYPE_BUL,
                                                 XUD_EPTYPE_BUL};
 
```

Comment:
Following on from Michael's observation about indentation, the above is similare...